Designaroni/designaroni-next
# feat(next-sitemap robots.txt, sitemap.xml)

Adds auto generated sitemap & robots files

![Replace me with a new gif](https://media.giphy.com/media/3orif8Drq5W7CQX6Hm/giphy-downsized.gif)

## Description

[//]: # 'Replace "JIRA-ISSUE" in the link below with the Jira Task ex: DN-000'

[JIRA ticket](https://designaroni.atlassian.net/browse/DN-59)

[//]: # "What changes are made in the pull request? minor, major & chore or cleanup tasks..."

## Documentation

[//]: # "Provide links to in-house documentation for this work or any third party documentation used"

## Impacted areas of code & app functionality

[//]: # "Describe impacted areas and areas affected by this change."

Adds `next-sitemap` to project to auto generate robotos.txt and sitemap.xml

## How to recreate & verify

[//]: # "Neurotically list steps needed to recreate the issue. Separately list steps to verify the fix or verify that all require changes satisfy the ticket requirements."

**Steps to verify:**

- start up the app...
- run `yarn build local`
- ensure `/robots.txt` and `/sitemap.xml` are up and available

## Checklists

### Pull Request Author

I have verified...


### Code Reviewer:

I have verified...
